en_US.json

```diff
@@ -244,6 +244,22 @@
 	"certificate.deploy.inprogress.modal.title": "Deploying Certificate",
 	"certificate.deploy.success.modal.message": "The certificate has been successfully deployed.",
 	"certificate.deploy.success.modal.title": "Certificate Deployed",
+	"certificate.dns01chanllenge.error.another.instance": "Our system is busy validating another request. Please try again in a few minutes.",
+	"certificate.dns01chanllenge.error.exist.certificate": "You have an existing certificate that has exactly the same domains or certificate name you requested and isn't close to expiry.",
+	"certificate.dns01chanllenge.error.exist.timeout": "Time out to wait. Please try again",
```

Comment:
"Time out to wait" sounds really strange. What do you mean here?
"Timeout" is a word, but "Timeout to wait" doesn't make sense.

Reply:
When issuing a certificate, waits for 30 minutes until clicking "Generate". But After that, if the user clicks "Generate", this message is displayed.

Reply:
I'm not sure I understand the first part. Are you saying the user has to wait 30 minutes until they can click the Generate button? That's a very long time.

Reply:
As we know, only one certbot can be run. Therefore, if there is no input after the message below is confirmed, it is set to terminate for the next user.
It says to wait up to 30 minutes, and the user can check and end at any time before that.

Please deploy a DNS TXT record under the name:
_acme-challenge.test.com.
with the following value:
VhtgnPIFy9ib0is2paJ8BpGSCgdfais-fUmaiD-7Y0c
You have 30 mins to finish this step...

Reply:
Are you saying the person ran out of the 30 minutes?
I suggest we be explicit and tell the user they have 30 minutes in advance.
For example:
"The setup of the DNS TXT records must be completed within 30 minutes. Please try again."

You can, of course, change it to an appropriate value if 30 is not correct.

"dns01chanllenge" should be "dns01challenge".

Reply:
I fixed some text and changed the language for ar_SA.
Thanks